### DISCUSSION TITLE
React 19 support

### DISCUSSION THREAD

**Discussion by puentesdiaz** [ai/nanostores#312] [General]
Is there a plan for React 19?

**Comment by ai** [comment]
Of course, we plan. It should just work.

What features do we need for React 19? Just update peer dependency?

**Reply by aminabromand** [reply]
- I assume "Just update peer dependency?", is all, that is needed.

I am currently using React 19 together with nanostores and all those functionalities, that I use, seem to work fine.
The only annoyance is the need for using `--legacy-peer-deps`.

**Comment by puentesdiaz** [comment]
I just trying use it via Astro, and there... package point to 18 and crash it
